src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -1599,8 +1605,8 @@ def pick_strategies(strategy_pool, fuzzer_path, corpus_directory,
   is_ephemeral = environment.is_ephemeral()
 
   if (not is_fuchsia and not is_android and not is_ephemeral and
-      (use_dataflow_tracing or
-       strategy_pool.do_strategy(strategy.FORK_STRATEGY))):
+      not use_dataflow_tracing and
```

Comment:
So, you don't want fork mode to be used in dataflow tracing anymore ?

Reply:
yes

Reply:
Could you add a comment explaining why they should be mutually exclusive here? Do they actually need to be mutually exclusive or could we still occasionally pick both?

Reply:
Done, also fixed a stale comment (that somehow went too above in the code), thanks!

The strategies are not mutually exclusive, but in the fork mode the log is pretty poor, so I want to get readable logs from the DFT runs for now. They include coverage information, focus function selected, and some other DFT-specific stats.